contents/handbook/people/onboarding.md

```diff
@@ -51,15 +51,44 @@ Every new joiner at PostHog has an onboarding buddy. If possible, a new joiner w
 - Please make sure you spend at least 3 days together, working through the [first week onboarding list](/handbook/people/onboarding#during-their-first-week) and spending time working on any role-specific tasks that are outlined in the new joiner's personal onboarding issue. 
 - You will remain the new joiner's main point of contact for the first few weeks, so please continue to check in with them at least once a week for the first month or so. 
 
-## In-person Onboarding 
+## In-person onboarding
```

Comment:
Are we linking to this from the onboarding GH issues already? If not we likely want to